src/common/websocket/api.ts

```diff
@@ -54,6 +54,7 @@ export default class DerivWS {
         const connection = new WebSocket(this.socket_url)
         connection.onopen = this.openHandler
         connection.onmessage = this.messageHandler
+        connection.onclose = this.reloadPage
```

Comment:
```suggestion
        connection.onclose = this.reloadHandler
```

Reply:
Made function recursive and depends on users activity. Please check again @prince-deriv 🙏